### PR TITLE
feat: add support for canonical formalism

### LINKED ISSUE #52: Add canonical formalism functionality
- [x] retrieve Clebsch Gordan factor from the recipe **[Estimated effort: 10min]**
- [x] evaluate the factor into a constant **[Estimated effort: 1h-1d]**
- [x] set L appropriately **[Estimated effort: 1min]**

### PR DESCRIPTION
**Pull request by spflueger**
* Add support for canonical sum from a helicity amplitude. This defines
the appropriate Clebsch Gordan coefficients.
* Add tests for canonical formalism parts.

Closes #52

### CODE REVIEW

File: tensorwaves/physics/helicityformalism/amplitude.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
 
 import numpy
 
+from sympy.physics.quantum.cg import CG
```
Comment (redeboer) [inline tensorwaves/physics/helicityformalism/amplitude.py:32]: Should be added to `requirements.txt` and `setup.py`, right?

File: tests/physics/helicityformalism/test_canonical.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                        "J": 1.0,
+                        "M": 1.0,
+                        "j1": 0.0,
+                        "m1": 0.0,
+                        "j2": 1.0,
+                        "m2": 1.0,
+                    }
+                },
+            },
+            math.sqrt(1 / 10) * 1,
```
Comment (redeboer) [inline tests/physics/helicityformalism/test_canonical.py:101]: Hmm class `dict` problem again... Keep thinking there's a case for https://github.com/ComPWA/expertsystem/issues/22

Comment (spflueger) [reply tests/physics/helicityformalism/test_canonical.py:101]: Hmm, but the current code just works with dicts? So this is unrelated comment right?

Comment (redeboer) [reply tests/physics/helicityformalism/test_canonical.py:101]: Yeah, it's not something for the review.

File: requirements.txt [hunk 3]
```diff
@@ -4,5 +4,6 @@ numpy
 phasespace
 progress
 pyyaml
+sympy
```
Comment (redeboer) [inline requirements.txt:7]: Should also be added to `setup.py` (which is used by PyPI)

File: requirements.txt [hunk 4]
```diff
@@ -4,5 +4,6 @@ numpy
...
 tensorflow==2.1.*
 xmltodict
```
Comment (redeboer) [inline requirements.txt:9]: `xmltodict` can indeed be removed (also from `setup.py`!)

File: requirements.txt (hunk 3 above)
Comment (spflueger) [reply requirements.txt:7]: 1bf2ee2ba9cb0527d925a6388fa2b82a835e51e2

File: requirements.txt (hunk 4 above)
Comment (spflueger) [reply requirements.txt]: 1bf2ee2ba9cb0527d925a6388fa2b82a835e51e2

**Comment by spflueger** [comment]
> Nice! Now all we need is to illustrate how to use all this in some notebooks (and ideally in the tests as to keep coverage high).

Yes, I think one the expertsystem output can be parsed directly we can start working on the notebooks etc

**Pull request merged**